test/unit/transient-error-lookup-test.js

```diff
@@ -0,0 +1,20 @@
+const TransientErrorLookup = require('../../src/transient-error-lookup').TransientErrorLookup;
+
```

Comment:
Makes it even more silly now that I changed the code to indexOf :-)
This is just an assertion to make sure any additions or deletion from list of transient errors is intentional.